### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Trust-007**
# Added testing
- Added testing for the operate and calculate functions
- Added testing for components
- Fixed linter errors

### CODE REVIEW

File: src/test/component.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  it('renders Quote component correctly', () => {
+    const tree = renderer.create(<QuoteContent />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('full app rendering/navigating', async () => {
+    render(<Layout />, { wrapper: BrowserRouter });
+    expect(screen.queryByText(/Home/)).toMatchSnapshot();
+  });
+});
```
Comment (TadesseDev) [inline src/test/component.test.js]: - [x] Good job @team. You did an impressive job already. One major thing left is for you to use [React Testing Library](https://github.com/testing-library/react-testing-library) to simulate user interaction. 
> Remember: this is different from `snapshots` tests. While `snapshots` tests are awesome to follow up when the UI changes the `React Testing Library` is used to simulate the user interaction. 
### Some examples would be 
```JavaScript
expect(screen.queryByText(testMessage)).toBeNull()
fireEvent.click(screen.getByLabelText(/show/i))
expect(screen.getByText(testMessage)).toBeInTheDocument()
```

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
EquatorialVictoriousBlackpanther-size_restricted

## Highlights
 - [ ] Good job overall writing the tests 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] I see your readme is not updated to have both collaborators. Pleas add both of your information to your repo. Its always recommended having all the collaborates mentioned in the README.md file.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/component.test.js (hunk 1 above)
Comment (Trust-007) [reply src/test/component.test.js]: Thanks for the review and suggestion @Tadesse-Alemayehu.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/test/component.test.js (hunk 1 above)
Comment (TadesseDev) [reply src/test/component.test.js]: Good job making the changes @Trust-007 👌

**Pull request merged**